core/templates/dev/head/collection_player/collection_player.html

```diff
@@ -0,0 +1,89 @@
+{% extends "base.html" %}
+
+{% block subtitle %}
+  {{ collection_title }}
+{% endblock subtitle %}
+
+{% block header_js %}
+  {{ super() }}
+  {% if collection_id %}
+    <script type="text/javascript">
+      GLOBALS.collectionId = JSON.parse('{{collection_id|js_string}}');
+      GLOBALS.isLoggedIn = Boolean(JSON.parse('{{is_logged_in|js_string}}'));
+    </script>
+
+    <style>
+      html, body {
+        background-color: #eee;
+      }
+    </style>
+  {% endif %}
+
+  {{dependencies_html}}
+{% endblock header_js %}
+
+{% block navbar_breadcrumb %}
+  <ul class="nav navbar-nav oppia-navbar-breadcrumb">
+    <li>
+      <span class="oppia-navbar-breadcrumb-separator"></span>
+      <span class="protractor-test-collection-header" itemprop="description">{{collection_title}}</span>
+    </li>
+  </ul>
+{% endblock navbar_breadcrumb %}
+
+{% block content %}
+  <div ng-controller="CollectionPlayer">
+    <div ng-if="collection" class="oppia-collection-player-tiles-area">
+      <div class="oppia-collection-player-tiles-container">
+        <div ng-if="!hasFinishedCollection()" class="oppia-collection-player-tiles-header">
+          <span ng-if="!hasStartedCollection()">Begin by playing:</span>
+          <span ng-if="hasStartedCollection()">Continue where you left off:</span>
+        </div>
+        <div ng-if="hasFinishedCollection()" class="oppia-collection-player-tiles-header">
+          <span>You have finished the collection! Feel free to replay any explorations below.</span>
+        </div>
+
+        {% if not is_logged_in %}
+          <span class="oppia-collection-player-small-text">You must login to save your progress.</span>
+        {% endif %}
+
+        <div ng-repeat="collection_node in getDisplayedCollectionNodes()">
+          <md-card layout="row" class="oppia-collection-player-tile protractor-test-collection-player-tile">
+            <a ng-href="/explore/<[collection_node.exploration.id]>?collection_id=<[collectionId]>" class="oppia-collection-player-tile-container-link" style="text-decoration: none;">
+              <div ng-class="{'oppia-collection-player-finished-exploration-tile': collection_node.exploration.completed}" class="oppia-collection-player-tile-contents">
+                <span class="oppia-collection-player-tile-image-container">
+                  <img class="oppia-collection-player-tile-image" ng-src="/images/gallery/exploration_background_teal_small.png">
+                </span>
+
+                <div class="oppia-collection-player-tile-details">
+                  <span class="oppia-collection-player-tile-title protractor-test-gallery-tile-title">
+                    <[collection_node.exploration.title]>
```

Comment:
I like the idea of revealing the newly unlocked explorations when you go back to the collections page (as well as at the end of the EndExploration in the recommended tiles area). I also agree everything being discussed here is out of scope for the MVP.

I just emailed in @xtyang and will hopefully have a chance to meet-up and sync.